packages/grafana-runtime/src/config.ts

```diff
@@ -124,6 +129,7 @@ export class GrafanaBootConfig implements GrafanaConfig {
   pluginAdminExternalManageEnabled = false;
   pluginCatalogHiddenPlugins: string[] = [];
   pluginCatalogManagedPlugins: string[] = [];
+  pluginsPreinstalled: PreinstalledPlugin[] = [];
```

Comment:
Nit: Worth updating for consistency, maybe?
```suggestion
  pluginCatalogPreinstalledPlugins: PreinstalledPlugin[] = [];
```